packages/terra-hyperlink/tests/wdio/hyperlink-spec.js

```diff
@@ -0,0 +1,120 @@
+/* global browser, Terra, before */
+const viewports = Terra.viewports('tiny', 'medium', 'large');
+
+describe('Hyperlink', () => {
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/default-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Disabled', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/disabled-hyperlink'));
+
+    const rules = {
+      'color-contrast': { enabled: false },
```

Comment:
Should we log an issue and add a TODO that reference here why we need to disable color-contrast?

Reply:
My understanding is UX has interpreted the [WCAG's 1.4.3 Contrast (Minimum) - Incidental: Text or images of text that are part of an inactive user interface component ... have no contrast requirement](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), to mean disabled UI does not need to meet contrast ratios and such, provide this color for the disabled state.

Need to check with @neilpfeiffer on this though, then we can add a comment in the code why we need to disable the linter here.

Reply:
Correct - inactive elements or incidentals generally we use a 2:1 ratio, rather than the traditional 4.5:1 for normal text or 3:1 for large text (14pt bold or 18pt normal or larger).

Reply:
Updated: https://github.com/cerner/terra-core/pull/1562#discussion_r193008539

Reply:
Hi, this may be a separate issue, but I believe that it is relevant as well:
As we use DialogModal, we found that the other components under the dialog modal become dithered(or dark) when the dialog pops up and that decreases the color contrast rate under 4.5:1.
I think that the DialogModal also has to have the separate ratio as well since we do not want them to be asserted with the decreased color contrast.